Development (#3)
NREL/PV_ICE: README.md

<img src="docs/images_wiki/CE-MFC.png" width="400">

<table>
<tr>
  <td>License</td>
  <td>
    <a href="https://github.com/NREL/CircularEconomy-MassFlowCalculator/blob/master/LICENSE">
    <img src="https://img.shields.io/pypi/l/pvlib.svg" alt="license" />
    </a>
</td>
</tr>
<tr>
  <td>Build Status</td>
  <td>
    <a href="http://circulareconomy-massflowcalculator.readthedocs.org/en/latest/">
    <img src="https://readthedocs.org/projects/circulareconomy-massflowcalculator/badge/?version=latest" alt="documentation build status" />
    </a>
  </td>
</tr>
<tr>
  <td>Publications</td>
  <td>
    <!--- <a href="https://doi.org/10.5281/zenodo.3762635">
    <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.3762635.svg" alt="zenodo reference">
     ---> Connected to Zenodo, awaiting first release
    </a>
  </td>
</tr>
</table>

> # CEMFC: Circular Economy - Mass Flow Calculator

Reviewer: If this tool is to forever be specific to PV, I suggest specifying that prominently, and even consider adding PV to the name of the package. `pv_mass_flow` or something. If it might conceivably go beyond PV someday, don't bother changing the name, but still mention the specific PV use case prominently in the readme. I'd lean toward leaving the door open to beyond-PV applications.

Author: Renamed to PV_ICE

> This open-source tool will help quantify and assign a value framework to efforts on re-design, reduction, replacement, reusage, recycling, and lifetime and reliability increases on PV, implementing Circular Economy metrics for PV materials.

Reviewer: ```suggestion
This open-source tool implements Circular Economy metrics for photovoltaic (PV) materials. It can be used to quantify and assign a value framework to efforts on re-design, reduction, replacement, reuse, recycling, and lifetime and reliability increases in the PV value chain.
```

Degradation rate (in percentage power loss per year) (D. C. Jordan, S. R. Kurtz, K. VanSant, and J. Newmiller, “Compendium of photovoltaic degradation rates,” Progress in Photovoltaics: Research and Applications, vol. 24, no. 7, pp. 978–989, 2016, doi: 10.1002/pip.2744.)

Glass thickness data (ITRPV, “International Technology Roadmap for Photovoltaic (ITRPV) 2019,” ITRPV, 10th, Oct. 2019.)

Reliability data, i.e. T50 and T90 lifetime in years (D. C. Jordan, B. Marion, C. Deline, T. Barnes, and M. Bolinger, “PV field reliability status—Analysis of 100 000 solar systems,” Progress in Photovoltaics: Research and Applications, vol. n/a, no. n/a, Feb. 2020, doi: 10.1002/pip.3262.)

All manufacturing, recyling, collection efficiencies currently estimates with real world data pending.

Installation
============

CircularEconomy-MassFlowCalculator releases may be installed using the ``pip`` and ``conda`` tools.
Please see the [Installation page](http://CircularEconomy-MassFlowCalculator.readthedocs.io/en/latest/installation.html) of the documentation for complete instructions.

CircularEconomy-MassFlowCalculator is compatible with Python 3.5 and above.

Install with:

>     pip install CEMFC

Author: @mdeceglie I'm confused about this comment because I included some scripts for it to pip install, and when I tested it in my computer  pip install CEMFC works. Have you tried it on your computer? Am I missing a step?  Please advise.

>     pip install CEMFC

Reviewer: Sorry, I should have been more specific. `pip install CEMFC` works for me, but only if I'm in the source directory. If the project is on pypi.org (e.g. https://pypi.org/project/bifacial-radiance/), then pip will go check pypi.org and download the package automatically no need to separately download the source. 

Here's the official tutorial: https://packaging.python.org/tutorials/packaging-projects/ 

I'd recommend getting it published on pypi once the release is ready.

For developer installation, download the repository, navigate to the folder location and install as:

    pip install -e .

> 

Author: Added intro tutorial
